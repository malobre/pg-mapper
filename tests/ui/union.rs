#[derive(pg_mapper::TryFromRow)]
union MyUnion {
    field: usize
}

fn main() {}
