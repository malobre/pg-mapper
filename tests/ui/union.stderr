error: `TryFromRow`: unions are not supported
 --> tests/ui/union.rs:2:1
  |
2 | / union MyUnion {
3 | |     field: usize
4 | | }
  | |_^
